File: bm_ui_android_kotlin/app/src/main/java/de/colinsteffen/bm_ui_android_kotlin/View/UIBenchmarkActivity2.java
package de.colinsteffen.bm_ui_android_kotlin.View;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.util.Log;
import android.widget.ListView;

import de.colinsteffen.bm_ui_android_kotlin.Helper.TimeHelper;
import de.colinsteffen.bm_ui_android_kotlin.R;
import de.colinsteffen.bm_ui_android_kotlin.View.Adapter.InformationAdapter;
import de.colinsteffen.bm_ui_android_kotlin.ViewModel.UIBenchmarkViewModel;

public class UIBenchmarkActivity2 extends AppCompatActivity{
    public UIBenchmarkActivity2(){
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_ui_benchmark2);
    }

    @Override
    protected void onResume(){
        super.onResume();

        TimeHelper.setEndTime();
    }
}

File: bm_ui_xam_platform/bm_ui_xam_platform/View/UIBenchmarkActivity2.cs
using System;
using System.Collections.Generic;
using System.Diagnostics;
using System.Linq;
using System.Text;

using Android.App;
using Android.Content;
using Android.OS;
using Android.Runtime;
using Android.Support.V7.App;
using Android.Views;
using Android.Widget;
using Shared.ViewModel;
using Shared.Helper;

namespace bm_ui_xam_platform.View
{
    [Activity(Label = "UIBenchmarkActivity2", NoHistory = true)]
    public class UIBenchmarkActivity2 : AppCompatActivity
    {
        protected override void OnCreate(Bundle savedInstanceState)
        {
            base.OnCreate(savedInstanceState);
            SetContentView(Resource.Layout.actvity_ui_benchmark2);
        }

        protected override void OnResume()
        {
            base.OnResume();

            TimeHelper.SetEndTime();
        }
    }
}